lms/djangoapps/certificates/models.py

```diff
@@ -191,6 +220,19 @@ class GeneratedCertificate(models.Model):
     created_date = models.DateTimeField(auto_now_add=True)
     modified_date = models.DateTimeField(auto_now=True)
     error_reason = models.CharField(max_length=512, blank=True, default='')
+    # Whether or not this GeneratedCertificate represents a
+    # certificate which can be shown to the user. Grading and
+    # certificate logic is intertwined here, so even enrollments
+    # without certificates (as of Jan 2016, this is only audit mode)
+    # create a GeneratedCertificate record to record the learner's
+    # final grade. Since audit enrollments used to have certificates
+    # and now do not, we need to be able to distinguish between old
+    # records and new in our analytics and reporting. The way we'll do
+    # this is by checking this field. By default it is True in order
+    # to make sure old records are counted correctly, and in
+    # `GeneratedCertificate.add_cert` we set it to False for new audit
+    # enrollments.
+    eligible_for_certificate = models.BooleanField(default=True)
```

Comment:
Can you catch up with Matt Piccicuto in Ed Svcs -- he has a spreadsheet that he uses to report on certificates which contains a column regarding eligibility status -- would be good to understand the criteria he uses to determine that (maybe it's the same, which would be nice)